src/config.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Node.js",
+  "description": "Node.js® is a JavaScript runtime built on Chrome's V8 JavaScript engine.",
+  "siteUrl": "https://nodejs.dev",
+  "siteUrlNoSlash": "https://nodejs.dev",
+  "featuredImage": "https://nodejs.org/static/images/logo-hexagon-card.png",
+  "color": "#6cc24a",
+  "icon": "src/images/favicon.png",
+  "lang": "en",
+  "display": "browser",
+  "ogType": "website",
+  "ogImgType": "image/png",
+  "ogImgWidth": "224",
+  "ogImgHeight": "256",
+  "twitter": "@nodejs",
+  "twitterCard": "summary",
+  "twitterImgAlt": "The Node.js Hexagon Logo",
+  "facebook": "https://www.facebook.com/nodejsfoundation/",
+  "bannerBtnText": "Blog post"
+}
```

Comment:
@manishprivet I removed the banners object from the config file. I only kept the button text because thats not provided by nodejs.org